File: leapyear.cpp
#include <iostream>
#include <conio.h>
using namespace std;

int main(int argc, char const *argv[])
{
	
	
	int year;

	cin>>year;

	if (year % 400 == 0 || year % 100!=0 && year % 4 == 0 ) // condition checks for a century year and not a century year but a leap year

		cout<<"Leap Year";
	else
		cout<<"Not Leap Year";
	

	return 0;
}

File: leapyear.py
#check leap year

x = int(input('Enter a year:'))

if x%400 == 0:                          #this condition checks if its a century year and a leap year
    print("leap year")
    
elif x%4 == 0 and x%100!= 0:            #checks if it is a leap year and not a century year
    print("leap year")
    
else:
    print("not a leap year")
